app/src/main/java/com/github/polypoly/app/MapActivity.kt

```diff
@@ -154,6 +161,48 @@ class MapActivity : ComponentActivity() {
         }
     }
 
+    @Composable
+    fun Hud(data: PlayerGlobalData, round: Int) {
```

Comment:
At a final stage of this feature, you should use current authenticated better than `data` argument. You could use `FirebaseAuth.getInstance().currentUser?.let{ //something }` (requires to add some dependencies), 
from which you can get the uid -> find the user data in data-base once it is all setup in Firebase-user-support PR.